src/OrleansVSTools/VSProjectSiloHost/VSProjectSiloHost.vstemplate

```diff
@@ -57,8 +57,8 @@
       <Asset Type="Microsoft.CodeAnalysis.Analyzers.1.0.0.nupkg" d:Source="File" Path="Packages\Microsoft.CodeAnalysis.Analyzers.1.0.0.nupkg" d:VsixSubPath="Packages" />
       <Asset Type="Microsoft.CodeAnalysis.Common.1.0.0.nupkg" d:Source="File" Path="Packages\Microsoft.CodeAnalysis.Common.1.0.0.nupkg" d:VsixSubPath="Packages" />
       <Asset Type="Microsoft.CodeAnalysis.CSharp.1.0.0.nupkg" d:Source="File" Path="Packages\Microsoft.CodeAnalysis.CSharp.1.0.0.nupkg" d:VsixSubPath="Packages" />
-      <Asset Type="Microsoft.Extensions.DependencyInjection.1.0.0.nupkg" d:Source="File" Path="Packages\Microsoft.Extensions.DependencyInjection.1.0.0.nupkg" d:VsixSubPath="Packages" />
-      <Asset Type="Microsoft.Extensions.DependencyInjection.Abstractions.1.0.0" d:Source="File" Path="Microsoft.Extensions.DependencyInjection.Abstractions.1.0.0" d:VsixSubPath="Packages" />
+      <Asset Type="Microsoft.Extensions.DependencyInjection.1.0.0-rc1-final.nupkg" d:Source="File" Path="Packages\Microsoft.Extensions.DependencyInjection.1.0.0-rc1-final.nupkg" d:VsixSubPath="Packages" />
+      <Asset Type="Microsoft.Extensions.DependencyInjection.Abstractions.1.0.0-rc1-final" d:Source="File" Path="Microsoft.Extensions.DependencyInjection.Abstractions.1.0.0-rc1-final" d:VsixSubPath="Packages" />
```

Comment:
It's the one that Orleans v1.2.3 uses. @attilah fixed the dependency here prematurely with what's on the master branch. Nevertheless this solution uses the officially released nuget  packages